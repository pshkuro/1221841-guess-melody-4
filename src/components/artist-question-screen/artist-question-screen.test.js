import React from "react";
import renderer from "react-test-renderer";
import ArtistQuestionScreen from "./artist-question-screen";

const question = {
  type: `artist`,
  song: {
    artist: `Jim Beam`,
    src: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
  },
  answers: [{
    picture: `https://api.adorable.io/avatars/128/0`,
    artist: `John Snow`,
  }, {
    picture: `https://api.adorable.io/avatars/128/1`,
    artist: `Jack Daniels`,
  }, {
    picture: `https://api.adorable.io/avatars/128/2`,
    artist: `Jim Beam`,
  }],
};

it(`Render ArtistQuestionScreen`, () => {
  const tree = renderer
  .create(
      <ArtistQuestionScreen
        question={question}
        onAnswer={() => {}}/>

  ).toJSON();

  // Add createNodeMock when Refs create

  expect(tree).toMatchSnapshot();
});
