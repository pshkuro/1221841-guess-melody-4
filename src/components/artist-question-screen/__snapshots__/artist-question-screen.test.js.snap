// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ArtistQuestionScreen 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Кто исполняет эту песню?
  </h2>
  <div
    className="game__track"
  >
    <div
      className="track"
    >
      <button
        className="track__button track__button--pause"
        disabled={false}
        onClick={[Function]}
        type="button"
      />
      <div
        className="track__status"
      >
        <audio />
      </div>
    </div>
  </div>
  <form
    className="game__artist"
  >
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-8879"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-8879"
      />
      <label
        className="artist__name"
        htmlFor="answer-8879"
      >
        <img
          alt="answer-8879"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/0"
        />
        John Snow
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-90"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-90"
      />
      <label
        className="artist__name"
        htmlFor="answer-90"
      >
        <img
          alt="answer-90"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/1"
        />
        Jack Daniels
      </label>
    </div>
    <div
      className="artist"
    >
      <input
        className="artist__input visually-hidden"
        id="answer-122"
        name="answer"
        onChange={[Function]}
        type="radio"
        value="answer-122"
      />
      <label
        className="artist__name"
        htmlFor="answer-122"
      >
        <img
          alt="answer-122"
          className="artist__picture"
          src="https://api.adorable.io/avatars/128/2"
        />
        Jim Beam
      </label>
    </div>
  </form>
</section>
`;
